message.go

```diff
@@ -0,0 +1,40 @@
+package git
+
+/*
+#include <git2.h>
+*/
+import "C"
+import (
+	"runtime"
+	"unsafe"
+)
+
+type Trailer struct {
+	key   string
+	value string
+}
+
+func MessageTrailers(message string) ([]Trailer, error) {
+
```

Comment:
minor nit:

```suggestion
```

(if this was copied from another file, let me know which one and we'll fix that later).